compute_sdk/globus_compute_sdk/sdk/executor.py

```diff
@@ -123,7 +123,7 @@ def __init__(
         label: str = "",
         batch_size: int = 128,
         funcx_client: Client | None = None,
-        amqp_port: int | None = None,
+        amqp_port: int = 443,
```

Comment:
Suggest keeping the `| None` typing, since we still have the code in place and documented that uses the port sent back from the web service